pom.xml

```diff
@@ -42,7 +42,7 @@
     <guava.version>18.0</guava.version>
     <slf4j.version>1.7.7</slf4j.version>
     <!-- plugin versions -->
-    <checkstyle.plugin.version>2.14</checkstyle.plugin.version>
+    <checkstyle.plugin.version>2.15</checkstyle.plugin.version>
```

Comment:
Actually, removing this line altogether would be a better option. The checkstyle version should be picked up from the fcrepo-parent pom.